config/initializers/node-orm2-connect.js

```diff
@@ -6,11 +6,16 @@ export default {
   before: 'define-orm-models',
   async initialize(application) {
     let container = application.container;
-    let adapter = container.lookup('orm-adapter:node-orm2');
-    let config = application.config.database;
+    let config = application.config;
+
+    if (!config.database || !config.database.orm2) {
+      // Config is not specified
+      return;
```

Comment:
Shouldn't we probably throw here? If you've included this addon but haven't configured database settings, something must be wrong, right?